ludwig/schema/combiners/base.py

```diff
@@ -4,4 +4,4 @@
 class BaseCombinerConfig(schema_utils.BaseMarshmallowConfig):
     """Base combiner config class."""
 
-    pass
+    type: str
```

Comment:
True, this is a good callout since there were a handful of changes that I just stumbled upon and made while I was at it, but each one of those increases the number of files and really isn't in scope. I will keep this in mind going forward!